test/Esprima.Tests.SourceGenerators/Snapshots/StringMatcherGeneratorTests.ScannerStringMatchingGeneration.01StringMatchers.g.verified.cs

```diff
@@ -350,31 +350,27 @@ public partial class Scanner
     {
         var c1 = input[1];
         var c2 = input[2];
```

Comment:
nit: `c2` is read unnecessarily when no branch matches `c1`. I'd also prefer nested switches here and then we wouldn't need these variables at all.